guides/delivery/team-changes-and-handovers.md

```diff
@@ -0,0 +1,64 @@
+
+### During the change
+#### Meet with the team
+Some of the team you’ll likely have worked with before, there may be others you haven’t. The sooner this happens the better. Take the time to speak with everyone in the team to understand what they are working on. The delivery lead will have confirmed with the team any personnel changes. The delivery lead/relevant Head Of should also confirm to the organisation about the team member change.
+
+#### Familiarise yourself with how the current project is set up
+You can speak with the delivery lead (or outgoing delivery lead if relevant) to understand which ceremonies happen and when. You should also make sure you know of any particular arrangements the team have, for example, remote working arrangements, the working patterns of team members and so on. Make sure you’re clear on what the client expectations are for this. Your delivery lead should ensure you are added to all the relevant calendar invites. 
+
+#### Meet the client
+As you’re onboarded to the new project, you should be introduced to whoever is the client representative. It’s important they know who you are and what role you do. How formal this process needs to be will depend on the nature of the project.
+
+### After the change
+#### Catch up with your relevant line manager or profession lead
+Moving to a new project can be challenging. Take the opportunity to catch up with your relevant line manager or profession lead to make sure it’s gone well and that you’re happy with the change.
+
+Below is a checklist of things that you/your delivery lead should complete when you’re moving on to a new project:
+- Familiarise yourself with the project through reading and conversations with the delivery lead
+- Meet with the delivery lead
+- Know what roles and responsibilities you’ll be taking on
+- Get added to the regular ceremonies:
+  - Standup
+  – Sprint planning meetings 
+  - Retrospectives
+  - Show and tells
+- Get added to relevant Slack channels
+- Get added to Trello boards for the project
+- Know where the Google Drive folder is and where key documentation lives
+- If the project is co-located at a client’s office, make sure you understand the building arrangements. Your delivery lead should help in getting you any required pass or key.
+- Get access to any technical resources or bespoke software the team are using
+- Meet the team / client representative
+
+## When a member of your sprint team is on support
+We’ve [changed the way we handle support] (http://playbook.dxw.com/#/guides/support-and-on-call), to ensure that there’s always a developer dedicated to support, and that the responsibility cycles round everyone on the development team at dxw.
```

Comment:
There should be no space between `]` and `(` for Markdown to work as intended. If you check the diff in rich diff mode (in the Files changed tab), it will show you what the resulting HTML would look like.
![Screenshot 2019-11-29 at 14 33 39](https://user-images.githubusercontent.com/579522/69875612-7e907b80-12b5-11ea-9cf1-a3156d887e43.png)